contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -5,19 +5,24 @@
 
 doc: This is the most general application definition for multidimensional photoelectron spectroscopy.
 category: application
-NXmpes:
+    NXmpes:
   (NXentry):
     title:
+      exists: recommended
     start_time(NX_DATE_TIME):
       doc: "Datetime of the start of the measurement."
+    end_time(NX_DATE_TIME):
+      exists: recommended
```

Comment:
see also my comments in the NXmpes_xps PR:
https://github.com/FAIRmat-NFDI/nexus_definitions/pull/30#discussion_r1192707961